src/Wizard/Wizard.js

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+
+import { Box, Typography } from "@mui/material";
+
+import PropTypes from "prop-types";
+
+/**
+ * The Wizard component allows you to create a multi-step form.
+ * It handles title and layout.
+ */
+export default function Wizard({ title, children }) {
+  return (
+    <Box
+      sx={{
+        display: "flex",
+        flexDirection: "column",
+        height: "100%"
+      }}
+    >
+      {title ? (
+        <Typography
+          variant="h5"
+          color="textPrimary"
+          sx={{
+            fontWeight: 500,
+            maxWidth: 945,
+            mb: 3,
+            mt: 1,
+            mx: "auto",
+            width: "100%"
+          }}
+        >
+          {title}
+        </Typography>
+      ) : null}
+      {children}
```

Comment:
I think wizard should only be accepting known children and also rendering them in the correct order like MUI dialog behaves. I don't see the order documented anywhere either.

If I change the order to this then I can break the component layout.
![image](https://user-images.githubusercontent.com/8976377/215430753-51b436eb-1f0b-4bbd-a3ab-043ceeb752b5.png)

![image](https://user-images.githubusercontent.com/8976377/215430815-cb60afa1-9045-4598-8d23-b83a2fe55309.png)

Reply:
I did actually play around with Dialog when writing this and there are no restrictions in place that I can tell. You can render the Dialog children in any order you see fit, no warnings.

I can certainly throw a warning if the children appear out of order.